apps/condo/domains/analytics/schema/ExternalReport.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * Generated by `createschema analytics.ExternalReport 'type:Select:metabase; title:Text; description?:Text; meta?:Json'`
+ */
+
+const { Text, Select, Checkbox, Virtual, Relationship } = require('@keystonejs/fields')
+const { Json } = require('@condo/keystone/fields')
+const { GQLListSchema } = require('@condo/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@condo/keystone/plugins')
+const conf = require('@condo/config')
+const { dvAndSender } = require('@condo/domains/common/schema/plugins/dvAndSender')
+const access = require('@condo/domains/analytics/access/ExternalReport')
+const get = require('lodash/get')
+const jwt = require('jsonwebtoken')
+const { OrganizationEmployee } = require('@condo/domains/organization/utils/serverSchema/index')
+
+
+const getMetabaseUrl = (externalReport, organizationId) => {
+    const METABASE_SITE_URL = conf['METABASE_SITE_URL']
```

Comment:
it's better to use one `METABASE_CONFIG` with `{"url":..., "key":...}` value

Reply:
Done